libs/entitlement/qa/src/javaRestTest/java/org/elasticsearch/entitlement/qa/EntitlementsTestRule.java

```diff
@@ -56,6 +56,7 @@ protected void before() throws Throwable {
             .systemProperty("es.entitlements.enabled", "true")
             .systemProperty("es.entitlements.testdir", () -> testDir.getRoot().getAbsolutePath())
             .setting("xpack.security.enabled", "false")
+            .setting("logger.org.elasticsearch.entitlement", "TRACE")
```

Comment:
Is this a leftover from testing/debugging or do you want to keep trace on?

Reply:
whoops